lib/router/router.js

```diff
@@ -276,7 +283,8 @@ export default class Router extends EventEmitter {
 }
 
 function getURL () {
-  return window.location.pathname + (window.location.search || '') + (window.location.hash || '')
+  const { href, origin } = window.location
+  return href.replace(new RegExp(`^${origin}`), '')
```

Comment:
yep, anyway I think we should avoid to create regexp from string as much as possible. It's slower, obscured and something incorrect. (we have to escape some chars like `.`)